_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,284 @@
+
+
+
+## Hands-on
+### Installation
+Just to try a few things, I installed Apiman (and the required server) as a standalone solution on to my laptop and used one of my earlier created API's to test.
+It was actually quite simple, however I did download a few versions of it, as some didn't work from the 1st minute.  
+One of the main advantages is that it has some pre-configured settings which made it easy to start.  
+Apiman is an Open Source tool, so a lot of documentation and even source code examples are available on the web.  
+If you prefer, it is also possible to dockerize it and run this in a container.  
+The latest version of Apiman can be found <a href="http://www.apiman.io" target="_blank" rel="noopener noreferrer">here</a>, or in case you want to try earlier versions or embedded in another server, you can also look in <a href="https://apiman.gitbooks.io/apiman-installation-guide/content/installation-guide/servlet/install.html#_installing_in_jboss_eap_7" target="_blank" rel="noopener noreferrer">this</a> link.  
+
+Once installed, I started to play around with it.  
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/APIMAN_Login.PNG" alt="APIMAN Login screen" width="600" height="375" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+### Setup
+I did some research on how to set it up, and on the key aspects of Apiman.  
+It's not my intention to go to much in dept of Apiman or the settings, but a few base principles I want to highlight to help you understand how easy it was to try it out and play with it.  
+The first very important part in Apiman was that everything starts with an Organization.  
+2ndly you need to setup a plan, which is basically a collection of policies that will be applied to requests made to Services being access through it.  
+As last you need to setup a Service. 
+A service contract is simply a link between an application and a service through a plan offered by that service. 
+When a service contract is created, the system generates a unique API key specific to that contract. 
+All requests made to the service through the API Gateway must include this API key.  
+
+Apiman start Screen.  
+<img src="/img/2020-09-14-Introduction-to-API-Management/APIMAN_Indexpage.PNG" alt="APIMAN start screen" width="600" height="375" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+Setting up an organization (OrgHome) and creating services.
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/APIMAN_Organization-API.PNG" alt="APIMAN Organization and Services" width="800" height="300" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+In this example you can see a Customer API which fetches Customer information from a Spring Boot application running in IntelliJ.  
+Every service can be marked public, so that everyone can access it, or you can for example secure it by creating a Client App and generate a required key via a plan.   

```

Comment:
```suggestion
Every service can be marked public, so that everyone can access it, or you can for example secure it by creating a client app and generate a required key via a plan.   

```